components/ipfs/ipfs_service.cc

```diff
@@ -405,9 +412,11 @@ void IpfsService::AddPin(const std::vector<std::string>& cids,
       requests_list_.insert(requests_list_.begin(), std::move(url_loader));
 
   iter->get()->Request(
-      "POST", gurl, std::string(), std::string(), false,
+      "POST", gurl, std::string(), std::string(),
       base::BindOnce(&IpfsService::OnPinAddResult, base::Unretained(this),
                      cids.size(), recursive, iter, std::move(callback)),
+      api_request_helper::APIRequestHelper::RequestOptions(false, -1u,
```

Comment:
Make use of https://google.github.io/styleguide/cppguide.html#Designated_initializers 
So RequestOptions{.timeout = base::Minutes(2)}